modules/javafx.graphics/src/main/java/javafx/animation/Interpolator.java

```diff
@@ -189,28 +189,57 @@ public String toString() {
     };
 
     /**
-     * Creates an {@code Interpolator}, which {@link #curve(double) curve()} is
+     * Use {@link #ofSpline(double, double, double, double)}.
```

Comment:
maybe something like "This poorly named method is deprecated in favor of ..." or words to that extent?

or keep the original description?